src/types/foliage.py

```diff
@@ -25,7 +25,7 @@ class TransactionsInfo(Streamable):
 @streamable
 class FoliageBlock(Streamable):
     # Information that goes along with each transaction block that is relevant for light clients
-    prev_block_hash: bytes32
+    prev_sub_block_hash: bytes32
```

Comment:
This is wrong. Foliage blocks have back pointers to the previous block, not the previous sub block

Reply:
indeed, copied from the validation code ...